docs/docsite/rst/getting_started/get_started_playbook.rst

```diff
@@ -0,0 +1,69 @@
+.. _get_started_playbook:
+
+*******************
+Creating a playbook
+*******************
+
+Playbooks are automation blueprints, in ``YAML`` format, that Ansible uses to deploy and configure managed nodes.
+Playbooks define the order in which Ansible performs actions, from top to bottom, and contain the following objects:
+
+Play
```

Comment:
You could mention above that a playbook is a list of plays (or maybe add a Playbook definition, along the lines of `An ordered list of plays to accomplish the overall goal of the playbook.`).

Reply:
Cool. I've reworked this to add a Playbook definition that incorporates the intro sentence.